fp.md

```diff
@@ -35,7 +38,11 @@ people.forEach(function(person) {
 });
 ```
 ```js
-// Your answer goes here
+people.filter(function(person) {
+  if(person.age < 35){
+    peopleYoungerThan35.push(person)
+  }
+});
```

Comment:
`.filter` also generates a new array. This time, it add a person to the resulting array if it meets whatever condition your returning in the code block.

```js
const peopleYoungerThan35 = people.filter(function(person){
  return person.age < 35 // If this expression evaluates to true for a particular person, that person will be added to the resulting array
})
```